deployment/terraform/cloud-config/ecs-batch-user-data.yml

```diff
@@ -0,0 +1,78 @@
+#cloud-config
+
+packages:
+    - awslogs
+
+fs_setup:
```

Comment:
That makes sense. There are some instances that automatically mount and format the devices. That's partially why `- [ /dev/nvme0n1, null ]` exists in the YAML (undo the default setups and redo them with custom options).